x/timerstore/keeper/grpc_query_next.go

```diff
@@ -0,0 +1,59 @@
+package keeper
+
+import (
+	"context"
+	"math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	commontypes "github.com/lavanet/lava/common/types"
+	"github.com/lavanet/lava/x/timerstore/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func createQueryNextResponse(nextBlockHeight, nextBlockTime uint64, tick string) *types.QueryNextResponse {
```

Comment:
tick is never used, needs to put in the tick

Reply:
Oops